src/pages/Wishlist/Wishlist.jsx

```diff
@@ -1,17 +1,33 @@
 import React from "react";
-import { Navbar, Footer } from "../../components";
+import { Navbar, Footer, ProductCard } from "../../components";
+import { useCartWishlist } from "../../context";
 import { useDocumentTitle } from "../../hooks";
+
 import("./wishlist.css");
 
 export function Wishlist() {
   useDocumentTitle("Wishlist");
+
+  const {
+    cartWishlist: { wishlist },
+  } = useCartWishlist();
   return (
     <div className='wishlist-container'>
       <Navbar />
       <main className='wishlist-main'>
         <h1 className='text-center wishlist-heading'>My Wishlist</h1>
-        <p className='md-ht-1 text-center'>You have 6 items in your wishlist</p>
-        <section className='wishlist-products'></section>
+        <p className='md-ht-1 text-center'>
+          You have {wishlist.length} items in your wishlist
```

Comment:
wishlist.length will only show the number of distinct products for eg if there are two products in cart with each quantity 2 this will only show two items in cart but there will be 4 items in total.

Reply:
I understand your point but in the cart, the aim is to only show a number of distinct items. Linking screenshot of flipkart's 
cart's page for more clarity on this
![Screenshot (662)](https://user-images.githubusercontent.com/28630412/160167972-8f54e14a-5c77-4b65-8930-7b34db1a5520.png)
Here you can see I have set the quantity to 4 but the items in the cart remain one only because that is what we want to show in the cart.